### PR TITLE
Fix an error in the total battery voltage display 

### PR DESCRIPTION
**Pull request by mxiaogit**
Fix an error in the total battery voltage display after more than 10 batteries.

### CODE REVIEW

**Comment by DonLakeFlyer** [comment]
@mxiaogit Thanks for the update

**Comment by mxiaogit** [comment]
@DonLakeFlyer Can this PR be merged?

**Review by patrickelectric — APPROVED** [review]
Looks good

File: src/Vehicle/VehicleBatteryFactGroup.cc [hunk 1]
```diff
@@ -142,6 +142,13 @@ void VehicleBatteryFactGroup::_handleBatteryStatus(Vehicle* vehicle, mavlink_mes
             totalVoltage += cellVoltage;
         }
     }
+    for (int i=0; i<4; i++) {
+        double cellVoltage = batteryStatus.voltages_ext[i] == UINT16_MAX ? qQNaN() : static_cast<double>(batteryStatus.voltages_ext[i]) / 1000.0;
```
Comment (patrickelectric) [inline src/Vehicle/VehicleBatteryFactGroup.cc:146]: it's only missing a const here, but should be not hold this PR.

**Pull request merged**